src/crawlee/basic_crawler/_basic_crawler.py

```diff
@@ -472,31 +472,53 @@ async def get_data(
         dataset = await Dataset.open(id=dataset_id, name=dataset_name)
         return await dataset.get_data(**kwargs)
 
-    async def export_data(
```

Comment:
As @vdusek said in the review message - we should probably keep this one with its limited interface and autodetection based on the path suffix.

Reply:
Hey @deshansh, we've been still waiting for this.